app/services/proofing_document_capture_session_result.rb

```diff
@@ -0,0 +1,59 @@
+class ProofingDocumentCaptureSessionResult
+  REDIS_KEY_PREFIX = 'dcs-proofing:result'.freeze
+
+  attr_reader :id, :pii, :result
+
+  class << self
```

Comment:
That is a great question and that's what felt weird when I was moving things over. I'm not sure because it's not in #4213 at all...?

I'll manually bring over #4217 into this PR

Reply:
@zachmargolis alright, fixed